kernel/dev/rtc/rtc.cpp

```diff
@@ -1,14 +1,17 @@
 /**
  * @file rtc.cpp
  * @author Keeton Feavel (keetonfeavel@cedarville.edu)
+ * @author Michel (JMallone) Gomes (michels@utfpr.edu.br)
  * @brief
- * @version 0.3
- * @date 2019-11-18
+ * @version 0.4
+ * @date 2021-07-25
  *
- * @copyright Copyright the Panix Contributors (c) 2019
+ * @copyright Copyright the Panix Contributors (c) 2021
  *
+ * References:
+ *          https://github.com/limine-bootloader/limine/blob/trunk/stage23/lib/time.c
```

Comment:
4 spaces instead of 8. 🙂

Reply:
@Jmallone Still have to do this one. Make sure the others are lined up too. 🙂 

This is the last issue! Then it'll be good to merge!